CMakeLists.txt

```diff
@@ -9,8 +9,8 @@ endif()
 
 project(${LIB_TARGET} LANGUAGES CXX VERSION 0.0.1)
 
-# require C++17
-set(CMAKE_CXX_STANDARD 17)
+# require C++20
```

Comment:
I am hesitant to switch to C++20. Unfortunately, it is still not so widely supported. I recently tried to use some C++20 feature only to learn that they are not available in the CI runners. Is there a particular reason to change the C++ verstion?

Reply:
I used `.contains` at some places. But I also can just do it with `.find`

Reply:
reverted back to 17